docscan/session/create/filter/document_filter.go

```diff
@@ -0,0 +1,47 @@
+package filter
+
+// RequestedDocumentFilter filters for a required document, allowing specification of restrictive parameters
+type RequestedDocumentFilter struct {
+	DocumentTypes []string `json:"document_types"`
+	CountryCodes  []string `json:"country_codes"`
+}
```

Comment:
This should only have `json:"type"`, and be embedded in `RequestedDocumentRestrictionsFilter` and `RequestedOrthogonalRestrictionsFilter`